migtests/scripts/functions.sh

```diff
@@ -893,6 +893,7 @@ normalize_json() {
             .OptimalSelectConnectionsPerNode? = "IGNORED" |
             .OptimalInsertConnectionsPerNode? = "IGNORED" |
             .RowCount? = "IGNORED" |
+			.FeatureDescription? = "IGNORED" | # Ignore FeatureDescription instead of fixing it in all tests since it will be removed soon
```

Comment:
for analyze-schema, it won't be removed, right? just for assess-migration.

Reply:
yes, for analyze schema, i have updated `Reason` field to match the new values